src/test/java/org/fundacionjala/coding/jorge/SortTwisted37Test.java

```diff
@@ -0,0 +1,38 @@
+package org.fundacionjala.coding.jorge;
+
+import org.junit.Test;
+import org.junit.Assert;
+
+/**
+ *
+ */
+public class SortTwisted37Test {
+
+    /**
+     *
+     */
+    @Test
+    public void test1() {
+        final Integer[] expected = new Integer[]{1, 2, 7, 4, 5, 6, 3, 8, 9};
```

Comment:
@AT-08/at-08 
the new Integer[] is redundant
```java
final Integer[] expected = {1, 2, 7, 4, 5, 6, 3, 8, 9};
```
Please apply to all your scenarios.